app/src/main/java/com/eveningoutpost/dexdrip/stats/StatsActivity.java

```diff
@@ -123,28 +126,61 @@ public void onPageScrollStateChanged(int state) {
     }
 
     void setButtonColors() {
-        buttonTD.getBackground().setColorFilter(null);
-        buttonYTD.getBackground().setColorFilter(null);
-        button7d.getBackground().setColorFilter(null);
-        button30d.getBackground().setColorFilter(null);
-        button90d.getBackground().setColorFilter(null);
-        switch (state) {
-            case TODAY:
-                buttonTD.getBackground().setColorFilter(new LightingColorFilter(0xFFFFFFFF, 0xFFAA0000));
-                break;
-            case YESTERDAY:
-                buttonYTD.getBackground().setColorFilter(new LightingColorFilter(0xFFFFFFFF, 0xFFAA0000));
-                break;
-            case D7:
-                button7d.getBackground().setColorFilter(new LightingColorFilter(0xFFFFFFFF, 0xFFAA0000));
-                break;
-            case D30:
-                button30d.getBackground().setColorFilter(new LightingColorFilter(0xFFFFFFFF, 0xFFAA0000));
-                break;
-            case D90:
-                button90d.getBackground().setColorFilter(new LightingColorFilter(0xFFFFFFFF, 0xFFAA0000));
-                break;
+
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
```

Comment:
shouldnt this be checking the version I am running rather than the build version? (or am I misreading this)